src/OSPSuite.R/RRegister.cs

```diff
@@ -44,7 +44,7 @@ public override void RegisterInContainer(IContainer container)
          container.Register<IOSPSuiteExecutionContext, RExecutionContext>(LifeStyle.Singleton);
          container.Register<IOSPSuiteLogger, RLogger, RLogger>(LifeStyle.Singleton);
          container.Register<IEventPublisher, EventPublisher>(LifeStyle.Singleton);
-         container.Register<ICsvSeparatorSelector, CsvSeparatorSelector>(LifeStyle.Singleton);
+         container.Register<ICsvDynamicSeparatorSelector, ICsvSeparatorSelector, CsvSeparatorSelector>(LifeStyle.Singleton);
```

Comment:
you need to Exclude from auto registration no?

Reply:
It was working fine without Excluding the type. I have just explicitly exclude it but not sure if necessary.

Reply:
oh it's in R only? I don't think there is automatic registration tehre

Reply:
oh yes there is. Then it would have created an issue when getting  ICsvSeparatorSelector (at least in theory)